conan/tools/system/pip_manager.py

```diff
@@ -23,6 +24,24 @@ def generate(self):
         env.prepend_path("PATH", self.bin_dir)
         env.vars(self._conanfile).save_script(self.env_name)
 
+    @staticmethod
+    def _default_python():
+        default_python = shutil.which('python') if platform.system() == "Windows" else shutil.which('python3')
+        return os.path.realpath(default_python) if default_python else None
+
+    def _create_venv(self):
+        python_interpreter = self._conanfile.conf.get("tools.system.pipenv:python_interpreter") or self._default_python()
+        if not python_interpreter:
+            raise ConanException("PipEnv could not find a Python executable path."
+                                 "Please set 'tools.system.pipenv:python_interpreter' to '</your/python/full/path>' "
+                                 "in the [conf] section of the profile, or in the command line using "
+                                 "'-c tools.system.pipenv:python_interpreter=</your/python/full/path>'")
+
+        try:
+            self._conanfile.run(cmd_args_to_string([python_interpreter, '-m', 'venv', self._env_dir]))
+        except ConanException:
+            raise ConanException("PipEnv could not create a Python virtual environment.")
```

Comment:
Maybe it's better if we do this?
```suggestion
        except ConanException as e:
            raise ConanException(f"PipEnv could not create a Python virtual environment using '{python_interpreter}': {e}")
```